app/src/main/res/values/strings.xml

```diff
@@ -3,50 +3,53 @@
     <string name="settings_backup_apk_dialog_cancel">Cancel</string>
-    <string name="settings_backup_apk_dialog_disable">Disable app backup</string>
+    <string name="settings_backup_apk_dialog_disable">Turn off app backup</string>
     <string name="settings_backup_status_title">Backup status</string>
     <string name="settings_backup_status_summary">Last backup: %1$s</string>
     <string name="settings_backup_exclude_apps">Exclude apps</string>
     <string name="settings_backup_now">Backup now</string>
     <string name="settings_category_storage">Storage backup (experimental)</string>
-    <string name="settings_backup_storage_title">Backup my files</string>
+    <string name="settings_backup_storage_title">Back up my files</string>
     <string name="settings_backup_files_title">Included files and folders</string>
     <string name="settings_backup_files_summary">None</string>
     <string name="settings_backup_recovery_code">Recovery code</string>
     <string name="settings_backup_recovery_code_summary">Verify existing code or generate a new one</string>
     <string name="settings_backup_storage_dialog_title">Experimental feature</string>
     <string name="settings_backup_storage_dialog_message">Backing up files is still experimental and might not work. Do not rely on it for important data.</string>
-    <string name="settings_backup_storage_dialog_ok">Enable anyway</string>
+    <string name="settings_backup_storage_dialog_ok">Turn on anyway</string>
     <string name="settings_backup_storage_battery_optimization">Warning: No automatic backups, because battery optimization is active.</string>
     <string name="settings_backup_new_code_dialog_title">New recovery code required</string>
     <string name="settings_backup_new_code_dialog_message">To continue using app backups, you need to generate a new recovery code.\n\nWe are sorry for the inconvenience.</string>
-    <string name="settings_backup_new_code_code_dialog_ok">New code</string>
+    <string name="settings_backup_new_code_code_dialog_ok">New recovery code</string>
 
     <string name="settings_expert_title">Expert settings</string>
     <string name="settings_expert_quota_title">Unlimited app quota</string>
     <string name="settings_expert_quota_summary">Do not impose a limitation on the size of app backups.\n\nWarning: This can fill up your storage location quickly. Not needed for most apps.</string>
-
+    <string name="settings_expert_logcat_title">Save app log</string>
+    <string name="settings_expert_logcat_summary">Sending this to the developers can help diagnose bugs.\n\nAlways review it to ensure it has no personally identifiable info, and delete it afterwards!</string>
+    <string name="settings_expert_logcat_error">Could not save app log</string>
+    
     <!-- Storage Location -->
     <string name="storage_fragment_backup_title">Choose where to store backups</string>
```

Comment:
```suggestion
    <string name="storage_fragment_backup_title">Choose where to save backups</string>
```
Perhaps less fluent, but store isn't different from "save" when it could be.
"Storage location" and "backup location" used elsewhere.